baseplate/sidecars/event_publisher.py

```diff
@@ -181,7 +192,7 @@ def publish_events() -> None:
         {
             "collector": {
                 "hostname": config.String,
-                "version": config.Optional(config.Integer, default=1),
+                "version": config.Optional(config.String, default="2"),
```

Comment:
This is a small functional change with the default value, but it seemed the default value would always cause an error (perhaps that is intended?).

Reply:
oops, a bad default from the days when v1 still existed. this looks good, thanks.